### DISCUSSION TITLE
New feature requests

### DISCUSSION THREAD

**Discussion by tkeiyama** [wantedly/frolint#942] [Ideas]
Frolintに関してのリクエストを集める場所です。

This is the place to gather requests that people want frolint to do.

ex. lintルールのxxxをdefaultでerrorにしたい。

**Comment by kobayang** [comment]
[import/order](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md) を適用したい

**Reply by kobayang** [reply]
適用したい理由としては、各自で import sort したりして無駄な diff が発生してレビューノイズになる。
これを防ぎたい。

 eslint に取り込んで全体適用したい。社内での設定例は以下。

```
"import/order": [
      "error",
      { "newlines-between": "always", "groups": ["builtin", "external", "parent", "sibling", "index"] }
    ]
```
